util/cmake/FindSeqAn.cmake

```diff
@@ -354,9 +355,14 @@ if (NOT _SEQAN_FIND_OPENMP EQUAL -1)
 endif ()
 
 if (OPENMP_FOUND)
-    if (COMPILER_IS_CLANG AND (_GCC_VERSION MATCHES "^37[0-9]$"))
+    if (COMPILER_CLANG AND (_GCC_VERSION MATCHES "^37[0-9]$"))
```

Comment:
Maybe we should change _GCC_VERSION to _COMPILER_VERSION or use "CMAKE_CXX_COMPILER_VERSION" directly?